staging/src/github.com/kubeedge/api/README.md

```diff
@@ -0,0 +1,10 @@
+# API
+Schema of the external API types that are served by KubeEdge.
+
+# Purpose
+This library is the canonical location of the KubeEdge API definition.It will be published separately to avoid diamond dependency problems for users who depend on multiple KubeEdge components.
+On the other hand, this library also provides available CRD informers/listers/clientsets and KubeEdge API documentation for users.
+
+# Where does it come from?
+api is synced from https://github.com/kubeedge/kubeedge/tree/master/staging/src/github.com/kubeedge/api.
```

Comment:
Sorry, I cannot get what meaning of this sentence.  Or it should be `API is synced from https://github.com/kubeedge/api`?

Reply:
This expression is similar to the expression in k8s https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/api#where-does-it-come-from. It means that our independent api repo is synchronized from the staging directory. You can also see similar expressions in mapper-framework https://github.com/kubeedge/kubeedge/tree/master/staging/src/github.com/kubeedge/mapper-framework#where-does-it-come-from